### PR TITLE
[MOV] Manufacturing: Restructure scope

### PR DESCRIPTION
**Pull request by hojo-odoo**
This PR restructures the docs within the Manufacturing scope, organizing them into four subcategories:

- Basic setup
- Advanced configuration
- Workflows
- Shop Floor

### CODE REVIEW

**Comment by StraubCreative** [comment]
FYI @odoo/sales-doc-review @odoo/ecommerce-doc-review @odoo/employees-doc-review this PR briefly edited some doc/ref tags in your documentation. No major changes 👍

**Comment by StraubCreative** [comment]
cc: @jcs-odoo

**Review by samueljlieber — COMMENTED** [review]
Hi @hojo-odoo, thanks for this restructuring PR! This is just a preliminary tech-review to offer some suggestions. 

I agree that a restructure is needed to remove the ambiguous _management_ directory. I agree with your restructure, and I think this lays a good foundation to futher enrich content on `manufacturing.rst` to link to _basic setup_, _advanced configuration_, _workflows_, and the _Shop Floor_ app.

A couple of technical points–
- redirects need to be added to `redirects/17.0.txt` for each moved or renamed file.
- I suggest removing the `:nosearch:` meta tag from `manufacturing.rst` (I think it is a good landing page if a user searches for "manufacturing".
- I also suggest moving `shop_floor_overview.rst` into `shop_floor.rst` to reduce another unnecessary directory.

Once the merge conflict is resolved, and redirects are added, I can give this PR a full technical review. Thank you! :)

**Review by jcs-odoo — COMMENTED** [review]
Hi @hojo-odoo and @StraubCreative !

Is there maybe another purpose/meaning behind "advanced configuration" or "basic setup"? Is the distinction mostly basic/advanced, or setup/configuration?

From my point of view, the "setup" and "configuration" titles aren't as clear as what's in "workflows". But I don't have any better suggestion right now, I'm afraid.

As @samueljlieber mentioned, we'll need the redirection rules. Also, the tag in the commit message and PR title should be MOV instead of REF.

I usually prefer to apply structure changes to all supported versions to avoid having differences between the links and the files across versions (think of the updated doc or ref links in other pages), which could cause small conflicts during fw-ports, but it seems to be a minor issue in this case as the conflicts would be super easy to fix.

Just giving my two cents, as Zachary pinged me, but it's up to you and the PO :)

Cheers, and have a great day!

**Comment by hojo-odoo** [comment]
Hey @jcs-odoo,

Thanks for taking the time to review this. I will add the redirects when I push the next update. 

The "Basic Setup" section includes docs with the information necessary to use the application on a basic level. They will be relevant to virtually everyone using the app, even those who don't need to use the more complicated features. The "Advanced Configuration" section includes docs detailing how to configure specialty features that will only be used by certain businesses. 

In the future, I will push any restructures to the earliest supported version, and then forward port from there.

I'm just waiting on one additional Manufacturing doc to finish merging, and then I will rebase this PR so it can be merged.

Thanks,
hojo

File: redirects/17.0.txt [hunk 1]
```diff
@@ -11,6 +11,23 @@ applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_
...
+applications/inventory_and_mrp/manufacturing/management/work_order_dependencies.rst applications/inventory_and_mrp/manufacturing/advanced_configuration/work_order_dependencies.rst         # management/* --> advanced_configuration/*
+applications/inventory_and_mrp/manufacturing/management/unbuild_orders.rst applications/inventory_and_mrp/manufacturing/workflows/unbuild_orders.rst                                        # management/* --> workflows/*
+applications/inventory_and_mrp/manufacturing/management/one_step_manufacturing.rst applications/inventory_and_mrp/manufacturing/basic_setup/one_step_manufacturing.rst                      # management/* --> basic_setup/*
+applications/inventory_and_mrp/manufacturing/management/two_step_manufacturing.rst applications/inventory_and_mrp/manufacturing/basic_setup/two_step_manufacturing.rst                      # management/* --> basic_setup/*
+applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst applications/inventory_and_mrp/manufacturing/basic_setup/three_step_manufacturing.rst                  # management/* --> basic_setup/*
```
Comment (samueljlieber) [inline redirects/17.0.txt:30]: Update the order to be [alphabetical](https://github.com/odoo/documentation/blob/17.0/redirects/MANUAL.md#how-do-i-create-a-redirect-rule) :)
```suggestion
applications/inventory_and_mrp/manufacturing/management/bill_configuration.rst applications/inventory_and_mrp/manufacturing/basic_setup/bill_configuration.rst                              # management/* --> basic_setup/*
applications/inventory_and_mrp/manufacturing/management/configure_manufacturing_product.rst applications/inventory_and_mrp/manufacturing/basic_setup/configure_manufacturing_product.rst    # management/* --> basic_setup/*
applications/inventory_and_mrp/manufacturing/management/kit_shipping.rst applications/inventory_and_mrp/manufacturing/advanced_configuration/kit_shipping.rst                               # management/* --> advanced_configuration/*
applications/inventory_and_mrp/manufacturing/management/manufacturing_backorders.rst applications/inventory_and_mrp/manufacturing/workflows/manufacturing_backorders.rst                    # management/* --> workflows/*
applications/inventory_and_mrp/manufacturing/management/one_step_manufacturing.rst applications/inventory_and_mrp/manufacturing/basic_setup/one_step_manufacturing.rst                      # management/* --> basic_setup/*
applications/inventory_and_mrp/manufacturing/management/product_variants.rst applications/inventory_and_mrp/manufacturing/advanced_configuration/product_variants.rst                       # management/* --> advanced_configuration/*
applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst applications/inventory_and_mrp/manufacturing/workflows/scrap_manufacturing.rst                              # management/* --> workflows/*
applications/inventory_and_mrp/manufacturing/management/split_merge.rst applications/inventory_and_mrp/manufacturing/workflows/split_merge.rst                                              # management/* --> workflows/*
applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst applications/inventory_and_mrp/manufacturing/advanced_configuration/sub_assemblies.rst                           # management/* --> advanced_configuration/*
applications/inventory_and_mrp/manufacturing/management/subcontracting.rst applications/inventory_and_mrp/manufacturing/workflows/subcontracting.rst                                        # management/* --> workflows/*
applications/inventory_and_mrp/manufacturing/management/two_step_manufacturing.rst applications/inventory_and_mrp/manufacturing/basic_setup/two_step_manufacturing.rst                      # management/* --> basic_setup/*
applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst applications/inventory_and_mrp/manufacturing/basic_setup/three_step_manufacturing.rst                  # management/* --> basic_setup/*
applications/inventory_and_mrp/manufacturing/management/unbuild_orders.rst applications/inventory_and_mrp/manufacturing/workflows/unbuild_orders.rst                                        # management/* --> workflows/*
applications/inventory_and_mrp/manufacturing/management/use_mps.rst applications/inventory_and_mrp/manufacturing/workflows/use_mps.rst                                                      # management/* --> workflows/*
applications/inventory_and_mrp/manufacturing/management/using_work_centers.rst applications/inventory_and_mrp/manufacturing/advanced_configuration/using_work_centers.rst                   # management/* --> advanced_configuration/*
applications/inventory_and_mrp/manufacturing/management/work_center_time_off.rst applications/inventory_and_mrp/manufacturing/workflows/work_center_time_off.rst                            # management/* --> workflows/*
applications/inventory_and_mrp/manufacturing/management/work_order_dependencies.rst applications/inventory_and_mrp/manufacturing/advanced_configuration/work_order_dependencies.rst         # management/* --> advanced_configuration/*
```

File: content/applications/inventory_and_mrp/manufacturing.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/manufacturing.rst:1]: I'd recommend removing the `:nosearch:` meta tag from the `manufacturing.rst` doc

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @hojo-odoo, nice job adding the redirects to this PR, I tested each one and confirmed they are valid. I have a small suggestion to update the order to be alphabetical per the convention.

Lastly, I'd still like to recommend moving `shop_floor_overview.rst`'s content into `shop_floor.rst` to reduce another unnecessary directory. This will still allow you to build out that app's scope further down the road. It is up to you 🙂

Please tag me for one last look once these small items are addressed, thank you!

**Comment by hojo-odoo** [comment]
Hey @samueljlieber,

This is ready for another look. I've updated the redirects and removed the meta tag from manufacturing.rst. As discussed, I've left the Shop Floor overview doc as-is.

Thanks,
hojo

**Review by samueljlieber — APPROVED** [review]
Nice job @hojo-odoo, lets merge this thing 🚀
.....
@robodoo r+

**Pull request closed without merging**